.github/workflows/container-validation-backends.yml

```diff
@@ -292,6 +292,171 @@ jobs:
           test_type: "e2e, gpu_1"
           platform_arch: ${{ matrix.platform.arch }}
 
+  deploy-test-fault-tolerance:
+    runs-on: cpu-amd-m5-2xlarge
+    if: needs.changed-files.outputs.has_code_changes == 'true'
+    needs: [changed-files, operator, vllm, trtllm, sglang]
+    permissions:
+      contents: read
+    strategy:
+      fail-fast: false
+      matrix:
+        test_scenario:
+          - vllm-disagg-prefill-tp-1-decode-tp-1-dp-2-decode_worker_pod
+          - trtllm-disagg-prefill-tp-1-decode-tp-1-dp-2-decode_worker_pod
+          - sglang-disagg-prefill-tp-1-decode-tp-1-dp-2-decode_worker_pod
+    name: deploy-test-fault-tolerance (${{ matrix.test_scenario }})
```

Comment:
After switching to a mapping, this will need to be updated, and also be much easier to read. Lets just use the framework name here.